src/components/Pages/Home.js

```diff
@@ -1,11 +1,13 @@
 import React, {Component} from 'react';
 import Header from '../Common/Header';
-import {image} from '../assets/img/header-bg.jpg';
+import image from '../assets/img/header-bg.jpg';
```

Comment:
Let's be more explicit with our variables names.

Reply:
That's the way that the facilitator declared it, but I'm gonna keep it in mind, thanks

Reply:
That sound more like an excuse.
It's not wrong to do it the way you were taught but it's better to build your own opinion.

Reply:
Done, that's the background image, so i have renamed it like that